packages/clerk-js/src/ui/components/UserProfile/ConnectedAccountsSection.tsx

```diff
@@ -61,7 +61,13 @@ export const ConnectedAccountsSection = withCardStateProvider(() => {
 
             return (
               <Action.Root key={account.id}>
-                <ProfileSection.Item id='connectedAccounts'>
+                <ProfileSection.Item
+                  id='connectedAccounts'
+                  sx={t => ({
+                    borderRadius: t.radii.$lg,
+                    ':hover': { backgroundColor: t.colors.$neutralAlpha50 },
```

Comment:
Can we avoid the duplication here by putting the styles inside `ProfileSection.Item` ?

Reply:
We can't move them in `ProfileSection.Item` as we use it for stuff that don't need the hover state and the border radius but we can create a new component or add a prop to add those styles.

Reply:
We can move them in and add a prop or make a small wrapper as you said. I prefer the wrapper tbh.

Reply:
The only thing I don't like about the wrapper is that it's can be a bit confusing as it would be the same thing, the hover state is like having a variant.